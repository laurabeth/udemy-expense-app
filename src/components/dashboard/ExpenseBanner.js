import React from "react";

const ExpenseBanner = () => <div>Expense Banner</div>;

export default ExpenseBanner;
