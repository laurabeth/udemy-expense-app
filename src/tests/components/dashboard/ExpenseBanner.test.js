/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import ExpenseBanner from "../../../components/dashboard/ExpenseBanner";

describe("expense banner", () => {
  it("renders", () => {
    const wrapper = shallow(<ExpenseBanner />);
    expect(wrapper).toMatchSnapshot();
  });
});
