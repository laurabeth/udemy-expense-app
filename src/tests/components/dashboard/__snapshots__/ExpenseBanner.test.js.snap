// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expense banner renders 1`] = `
<div>
  Expense Banner
</div>
`;
